scrapy/cmdline.py

```diff
@@ -32,7 +32,7 @@ class ScrapyArgumentParser(argparse.ArgumentParser):
     def _parse_optional(
         self, arg_string: str
     ) -> tuple[argparse.Action | None, str, str | None] | None:
-        # if starts with -: it means that is a parameter not a argument
+        # if starts with -: it means that it is a parameter not an argument
         if arg_string[:2] == "-:":
```

Comment:
And you could also take the chance to switch this to `startswith()`.

```suggestion
        if arg_string.startswith("-:"):
```

@wRAR I checked to see if there was something in Ruff that we could enable to detect these cases, but it seems not. They may add it if they ever [port Sonar rules](https://github.com/astral-sh/ruff/issues/4935).